app/controllers/friendships_controller.rb

```diff
@@ -3,10 +3,9 @@ def create
     friend = User.find_by(email: params[:email])
     if friend
       Friendship.create_reciprocal_for_ids(current_user.id, friend.id)
-      redirect_to "/dashboard"
     else
-      flash[:errors] = "Friend does not exist in system. Cannot add."
-      redirect_to "/dashboard"
+      flash[:errors] = 'Friend does not exist in system. Cannot add.'
     end
+    redirect_to '/dashboard'
```

Comment:
Rubo cop is so helpful! These little technical mistakes would be so hard to catch without it.